src/Components/Publishing/Series/Components/ArticleCard.tsx

```diff
@@ -42,7 +42,7 @@ export class ArticleCard extends Component<Props, null> {
 
         <ImageContainer>
           <Image src={crop(article.thumbnail_image, { width: 680, height: 450 })} />
-          {video &&
+          {media &&
             <VideoPlay><IconPlayCaret color={color} /></VideoPlay>
           }
         </ImageContainer>
```

Comment:
We've been putting our subcomponents in the main entity's directory, like with:

`Components/Publishing/Display/Canvas/CanvasText.tsx`
and
`Components/Publishing/Header/ClassicHeader.tsx`

It's not a big deal, but I'm wondering why you want to nest here with an additional `Components` directory? It looks like there are only 3 subcomponents and they all have good naming.